pkgs/applications/science/math/maxima/default.nix

```diff
@@ -93,7 +95,7 @@ stdenv.mkDerivation (finalAttrs: {
   #
   # These failures don't look serious. It would be nice to fix them, but I
   # don't know how and probably won't have the time to find out.
-  doCheck = false;    # try to re-enable after next version update
+  doCheck = true;    # try to re-enable after next version update
```

Comment:
Is this why this is a draft? 1 out of 1 tests failed so it seems according to the build log...

Reply:
No I was just careless staging my hunks. Removed.